pxf/build.gradle

```diff
@@ -286,6 +286,29 @@ project('pxf-hive') {
     }
 }
 
+project('pxf-json') {
+	dependencies {
+		compile(project(':pxf-hdfs'))
+		compile "org.apache.commons:commons-lang3:3.0"
+
+		testCompile 'pl.pragmatists:JUnitParams:1.0.2'
+		configurations {
+			// Remove hive-exec from unit tests as it causes VerifyError
+			testRuntime.exclude module: 'hive-exec'
```

Comment:
@hornn What do you men? do we need the pxf-json project? or do we need some of the dependencies/configurations?